README.md

```diff
@@ -41,11 +41,11 @@ Integration into home automation systems for intelligent automation, etc
 ### Block diagram
 
 <!-- Make sure the path to the picture is correct -->
-![Block Diagram](schematics/block_diagram.png)
+![Block Diagram](schematics/block_diagram.png) https://www.tinkercad.com/things/2SMvaSbqaCq-mon-projet
 
 ### Schematic
 
-![Schematic](schematics/kicad_schematic.png)
+![Schematic](schematics/kicad_schematic.png) https://www.tinkercad.com/things/2SMvaSbqaCq-mon-projet
 
```

Comment:
Please add a photo with the schematic. You can also keep the link to kicad if you like. But I want the image to be displayed directly in the documentation. Add the image to your repository and link it in the README.md file.